### PR TITLE
Bring main repo up to speed

### PR DESCRIPTION
**Pull request by philerooski**

### CODE REVIEW

File: R/clean_curated_data.R [hunk 1]
```diff
@@ -5,63 +5,178 @@ fetch_syn_table <- function(syn_id, cols = "*") {
   if (is.vector(cols)) {
     cols <- paste(cols, collapse = ",")
   }
-  table <- synapser::synTableQuery(paste("select", cols, "from", syn_id))
-  table_df <- table$asDataFrame()
+  table <- synTableQuery(paste("select", cols, "from", syn_id))
+  table_df <- table$asDataFrame() %>%
+    as_tibble()
+  if (has_name(table_df, "ObjectTrackID")) {
```
Comment (kdaily) [inline R/clean_curated_data.R]: This is duplicated in two places (all 'if's). Can it be abstracted to a function that mutates the data frame?

File: R/clean_curated_data.R [hunk 2]
```diff
@@ -5,63 +5,178 @@ fetch_syn_table <- function(syn_id, cols = "*") {
...
-get_potential_errors_corrected <- function() {
-  potential_errors_corrected <- synGet("syn18134075")$path %>%
-    read_csv() %>%
-    as_tibble() %>%
-    select(Experiment, ObjectTrackID, Well, TimePoint, Live_Cells, Lost_Tracking) %>%
-    mutate(ObjectTrackID = as.integer(ObjectTrackID),
-           TimePoint = as.integer(TimePoint))
+fill_in_missing_timepoints <- function(curated_cell_data) {
+  timepoint_reference <- fetch_syn_table(
+    "syn11817859", c("Experiment", "TimePointBegin", "TimePointEnd")) %>%
```
Comment (kdaily) [inline R/clean_curated_data.R]: pass Syn id as a parameter

Comment (kdaily) [reply R/clean_curated_data.R]: why is fxn duplicated here and in R/curate_curated_cell_table.R?

File: R/clean_curated_data.R (hunk 1 above)
Comment (philerooski) [reply R/clean_curated_data.R]: Yes. This is just sloppy coding on my part.

File: R/clean_curated_data.R (hunk 2 above)
Comment (philerooski) [reply R/clean_curated_data.R]: `R/curate_curated_cell_table.R` is an old version of `R/clean_curated_data.R`. For some reason I decided to nuke `R/curate_curated_cell_table.R` and start over. I'll take it out.

**Comment by philerooski** [comment]
@kdaily I'll make these changes and if you still want to have a meeting to go over the PR we can keep it on the calendar. I mostly wanted to get the cwl files up to date, but since there hasn't been a merge since February this PR includes the files to construct the cleaned curated "gold standard" dataset as well.

**Comment by kdaily** [comment]
Let's meet to discuss, but I think it's probably g2g - just want to be sure I understand it. Thanks!

File: workflows/scoring/score_per_object.cwl [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+#
+# Example score submission file
+#
+cwlVersion: v1.0
+class: CommandLineTool
+baseCommand: tracking_summary_perfect_tracks.R
+
+hints:
+  DockerRequirement:
+    dockerImageId: localneurolincsscoring
```
[the remark this answers is not in the document]
Comment (kdaily) [reply workflows/scoring/score_per_object.cwl]: Use `v0.2` and just assume it's going to be there

File: workflows/scoring/download_submission_file.cwl [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+            if __name__ == "__main__":
+                main()
+     
+outputs:
+  - id: filepath
+    type: File
+    outputBinding:
+      glob: $("submission-"+inputs.submissionId)
+  - id: entity
+    type:
```
[the remark this answers is not in the document]
Comment (philerooski) [reply workflows/scoring/download_submission_file.cwl]: It doesn't work without it so I suppose so.

[the remark this answers is not in the document]
Comment (thomasyu888) [reply workflows/scoring/download_submission_file.cwl]: Ya. Its part of CWL spec

File: workflows/scoring/score_email.cwl [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+#
+# Example score emails to participants
+#
+cwlVersion: v1.0
+class: CommandLineTool
+baseCommand: python3.6
+
+hints:
+  DockerRequirement:
+    dockerPull: sagebionetworks/synapsepythonclient
```
[the remark this answers is not in the document]
Comment (philerooski) [reply workflows/scoring/score_email.cwl]: hmm, this script will not work with the only tagged version available (1.9.2). I've asked Ziming to tag :latest with v1.9.3 https://sagebionetworks.jira.com/browse/PLFM-5922

**Pull request merged**